iree/compiler/Codegen/Common/LinalgBufferizePass.cpp

```diff
@@ -1546,6 +1568,13 @@ void LinalgBufferizePass::runOnOperation() {
           }
           return convertAnyLinalgOp(b, linalgOp, bvm, plan, allocationFn);
         })
+        .Case<linalg_ext::LinalgExtOp>([&](linalg_ext::LinalgExtOp op) {
```

Comment:
You could just make the `.Case` line at 1564 to `.Cast<linalg::LinalgOp, linalg_ext::LinalgExtOp>`

Reply:
This is great, thanks!